google/cloud/firestore_v1/query.py

```diff
@@ -729,13 +761,35 @@ def _to_protobuf(self):
         return query_pb2.StructuredQuery(**query_kwargs)
 
     def get(self, transaction=None):
-        """Deprecated alias for :meth:`stream`."""
-        warnings.warn(
-            "'Query.get' is deprecated:  please use 'Query.stream' instead.",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        return self.stream(transaction=transaction)
+        """Read the documents in the collection that match this query.
+
+        This sends a ``RunQuery`` RPC and returns a list with documents
+        returned in the stream of ``RunQueryResponse`` messages.
+
+        Args:
+            transaction
+                (Optional[:class:`~google.cloud.firestore_v1.transaction.Transaction`]):
+                An existing transaction that this query will run in.
+
+        If a ``transaction`` is used and it already has write operations
+        added, this method cannot be used (i.e. read-after-write is not
+        allowed).
+
+        Returns:
+            list: The documents in the collection that match this query.
+        """
+        if self._limit_to_last:
+            self._limit = self._limit_to_last
+
+            for order in self._orders:
+                order.direction = _enum_from_direction(
+                    self.DESCENDING
+                    if order.direction == self.ASCENDING
+                    else self.ASCENDING
+                )
+
+        result = list(self.stream(transaction=transaction))
+        return list(reversed(result))
```

Comment:
We can't use `sort()` in here, 'cause there is no comparison methods in document objects. So we're using `reversed()` to create a reversed iterator, and converting it to `list()`.

Reply:
:+1:  Can we propose a comparison method in a separate PR, what do you think?

Reply:
I'll ask Ben. I don't think they actually want it, as I saw some time ago a PR from a user with such a thing, and they closed it